src/adapters/philips_hue/api/mod.rs

```diff
@@ -182,6 +182,12 @@ impl HueLight {
         api.get_light_status(&self.hue_id)
     }
 
+    pub fn get_unique_id(&self) -> String {
```

Comment:
It doesn't *depend* on storage. From what I understoo, @npark-mozilla would benefit to have a stable id for his tests. No storage needed for that right?